README.md

```diff
@@ -90,6 +90,20 @@ make test
 sudo make install
 ```
 
+The following options can be added to the `cmake` call to control PNGwriter features:
+
+| CMake Option           | Values           | Description                             |
+|------------------------|------------------|-----------------------------------------|
+| PNGwriter_USE_FREETYPE | **AUTO**/ON/OFF  | Enable support for text via Freetype    |
```

Comment:
One suggestion and one nitpick:
* How about sorting the rows alphabetically?
* "Freetype" is spelled "FreeType"

Reply:
@rakuco no prob, updated! thx!